endpoints/getting-started-grpc/README.md

```diff
@@ -0,0 +1,139 @@
+# Google Cloud Endpoints Sample for Go using gRPC
+
+This sample demonstrates how to use Google Cloud Endpoints using Go and gRPC.
+
+## Test the code locally (optional)
+
+Run the backend using `go run`:
+
+```bash
+$ go run server/main.go
+```
+
+Sent a request:
+
+```bash
+$ go run client/main.go
+2017/03/30 17:08:32 Greeting: Hello world
+```
+
+## Deploying service config
+
+1. First, generate `out.pb` from the proto file:
+
+    ```bash
+    protoc --include_imports --include_source_info helloworld/helloworld.proto --descriptor_set_out out.pb
+    ```
+
+1. Edit `api_config.yaml`. Replace `YOUR_PROJECT_ID`:
+
+    ```yaml
+    name: hellogrpc.endpoints.YOUR_PROJECT_ID.cloud.goog
+    ```
+
+1. Deploy your service:
```

Comment:
As above, this is a cut-down version of the full guides:
https://cloud.google.com/endpoints/docs/quickstart-grpc-compute-engine-docker